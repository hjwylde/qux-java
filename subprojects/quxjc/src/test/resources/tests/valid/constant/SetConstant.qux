{int} A is {5, 4}
{int} B is {5, 4}
{int} C is {-5}

{int} f({int} a):
    return a + A

{int} g({int} a):
    return a


void main([str] args):
    print A
    print B
    print C
    print A == B
    print A == C
    print B == C
    print f(C)
    print g(C)

