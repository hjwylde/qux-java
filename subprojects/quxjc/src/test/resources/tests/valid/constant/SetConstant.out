{4, 5}
{4, 5}
{-5}
true
false
false
{-5, 4, 5}
{-5}
